skietbaan_fe/src/components/NavbarMenu.js

```diff
@@ -17,7 +16,7 @@ import { Navbar,
   <Navbar.Collapse id="basic-navbar-nav">
     <Nav className="mr-auto">
       <Nav.Link href="/home">Leader Board</Nav.Link>
-      <Nav.Link href="/login">Score Capture</Nav.Link>
+      <Nav.Link href="/Scores">Score Capture</Nav.Link>
       <Nav.Link href="/login">Notifications</Nav.Link>
```

Comment:
Notifications and documents redirect you to the same location.
Is it intentional?